src/omnicore/notifications.h

```diff
@@ -3,26 +3,48 @@
 
 #include <stdint.h>
 #include <string>
+#include <vector>
+
+//! Alert types
+#define ALERT_FEATURE_UNSUPPORTED     0 // not allowed via alert broadcast
```

Comment:
Was there a specific reason to use `#define` instead of `const uint16_t`? Just thinking, because `uint16_t` is also the native type used for the messages.